chart/helm-operator/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 appVersion: "1.0.0-rc3"
-version: 0.2.1
+version: 0.3.0
```

Comment:
Please undo this change, the version will be bumped when we do a Helm-op release